routers/api/v1/api.go

```diff
@@ -985,6 +985,11 @@ func Routes(ctx gocontext.Context) *web.Route {
 						}, mustEnableAttachments)
 					})
 				}, mustEnableIssuesOrPulls)
+				m.Group("/projects", func() {
+					m.Combo("").
+						Get(reqToken(auth_model.AccessTokenScopeRepo), repo.ListRepositoryProjects).
```

Comment:
I've used Repo Scope for project boards, but this probably needs to be reviewed as to which scope is most appropriate.

Reply:
Are [these](/go-gitea/gitea/blob/main/docs/content/doc/developers/oauth2-provider.en-us.md) all possible Scopes? There is nothing else except `AccessTokenScopeRepo` to fit then